UnityProject/Assets/Scripts/Player/PlayerNetworkActions.cs

```diff
@@ -13,6 +13,16 @@ public partial class PlayerNetworkActions : NetworkBehaviour
 	private static readonly StandardProgressActionConfig CPRProgressConfig =
 		new StandardProgressActionConfig(StandardProgressActionType.CPR);
 
+	/// <summary>
+	/// Fire stacks removed when patting fires (on other people)
+	/// </summary>
+	private readonly float patFireStacksRemoved = 4f;
```

Comment:
this stuff is really specific to fire-patting so it'd be preferable to define these in the fire patting component